doc/reference/tooling/tt_cli/import.rst

```diff
@@ -134,14 +149,66 @@ To skip rows whose data cannot be parsed correctly, use the ``--on-error`` optio
     $ tt crud import localhost:3301 customers.csv:customers \
                      --on-error skip
 
+.. _tt-import-tdg2:
+
+Importing into Tarantool Data Grid 2
+------------------------------------
+
+To import data into TDG2, prepare a JSON lines file with an input data and call
+``tt tdg2 import`` using this file as an input:
+
+.. code-block:: console
+
+    $ tt tdg2 import localhost:3301 customers.jsonl:customers
+
+The input file can look like this:
+
+.. code-block:: json
+
+    {"age":30,"first_name":"Samantha","id":1,"second_name":"Carter"}
+    {"age":41,"first_name":"Fay","id":2,"second_name":"Rivers"}
+    {"age":74,"first_name":"Milo","id":4,"second_name":"Walters"}
+
+.. note::
+
+    Since JSON describes objects in maps with string keys, there is no way to
+    import a field value that is a map with a non-string key.
+
+If an error happens during TDG2 import, the all the changes made within the current batch
```

Comment:
Well, it's not true (as well as it's not true for `tt crud import`). Rollback happens per-storage, not per-cluster. You can actually get (and are likely to get) half batch applied and half batch rolled back.